### PR TITLE
Support wasm parser

### PR DESCRIPTION
**Pull request by osipov-mit**

### CODE REVIEW

File: js/src/parser/parser.wasm
Comment (DennisInSky) [inline js/src/parser/parser.wasm:1]: Why do we want to publish it onto git? Isn't it just a build artifact?

**Review by DennisInSky — APPROVED** [review]

File: js/src/parser/parser.wasm
Comment (osipov-mit) [reply js/src/parser/parser.wasm:1]: no need

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+build:
```
[the remark this answers is not in the document]
Comment (osipov-mit) [reply Makefile]: renamed

**Review by DennisInSky — COMMENTED** [review]
1. Generated client should be able to pass some value with every single call to a program.
2. There should be 2 promises whilest calling a program:
    - first one for sending a call message to the system, aka publishing the message
    - second one for receving the actual program response after it process the message

**Review by DennisInSky — APPROVED** [review]

**Pull request merged**